test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -607,39 +613,33 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 
 		It("[test_id:50964] Offline Higher CPUID's", func() {
 			var reserved, isolated, offline []string
-			// This map is of the form numaNode[core][cpu-siblings]
```

Comment:
this comment should have been preserved when moving the var up the flow. But we can't fix into a backport